pom.xml

```diff
@@ -64,7 +64,7 @@
 
         <sonar-packaging.version>1.23.0.740</sonar-packaging.version>
 
-        <!-- temporary version waiting for real automatic release in ecocode repository -->
+        <!-- temporary version waiting for real automatic release in creedengo repository -->
         <ecocode-rules-specifications.version>1.6.2</ecocode-rules-specifications.version>
```

Comment:
please change this property name to "creedengo-rules-specifications" and use 2.0.0 version